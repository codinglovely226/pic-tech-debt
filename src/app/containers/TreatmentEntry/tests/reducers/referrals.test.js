/* eslint-disable */

import ReferralsReducer from '../../reducers/referrals';

import {
  updateReferrals,
} from '../../actions/case';

describe('ReferralsReducer', () => {
  it('returns the initial state', () => {
    expect(ReferralsReducer(undefined, {})).toMatchSnapshot();
  });
  it('update referrals', () => {
    var mockedDate = new Date(2018, 3, 7, 17, 8);
    global.Date = jest.fn(() => mockedDate);
    expect(ReferralsReducer([], updateReferrals(1, 'Test Referee', 'Test Specialty', 2))).toMatchSnapshot();
  });
});
