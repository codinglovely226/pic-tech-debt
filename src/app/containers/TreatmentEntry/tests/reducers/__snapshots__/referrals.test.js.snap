exports[`ReferralsReducer returns the initial state 1`] = `Array []`;

exports[`ReferralsReducer update referrals 1`] = `
Array [
  Object {
    "referee": "Test Referee",
    "refereeSite": 2,
    "specialty": "Test Specialty",
    "timestamp": "2018-04-07T17:08:00.000Z",
  },
]
`;
